docs/antrea-agent-simulator.md

```diff
@@ -0,0 +1,48 @@
+# Run Antrea agent simulator
+
+## 1. build the simulator image
+```bash
```

Comment:
I remember there are some format requirements for the doc to be rendered on website correctly.
For example, there needs to be an empty line before the code. Not sure there's CI job checking this. But you can follow other docs' style or standard markdown guide.

Reply:
there is a CI job and looks like it's failing on this PR, which is a good sign :)